File: src/main/java/com/lord/small_box/dtos/PurchaseOrderDto.java
package com.lord.small_box.dtos;

import java.math.BigDecimal;
import java.util.Calendar;
import java.util.List;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PurchaseOrderDto {
	private Long id;

	private int orderNumber;

	private String jurisdiction;

	private String executerUnit;
	
	private String financingSource;

	private String dependency;
	
	private String provider;

	private String deliverTo;

	private Calendar date;

	private String exp;
	
	private BigDecimal purchaseOrderTotal;

	private List<PurchaseOrderItemDto> items;
}

File: front/small-box/src/app/models/purchaseOrderDto.ts
import { PurchaseOrderItemDto } from "./purchaseOrderItemDto";


export class PurchaseOrderDto {
    id!: number;
    orderNumber!: number;
    jurisdiction!: string;
    executerUnit!: string;
    executerUnitOrganizationId!: number;
    financingSource!: string;
    dependency!: string;
    dependencyOrganizacionId!: number;
    provider!: string;
    deliverTo!: string;
    date!: Date;
    exp!: string;
    purchaseOrderTotal!: number;
    items: Array<PurchaseOrderItemDto> = [];
    loadedToDeposit!: boolean;
    loadedToDepositId!: number;
    loadedToDepositName!: string;
}